tests/ra_aid/proc/test_interactive.py

```
@@ -1,6 +1,7 @@
 """Tests for the interactive subprocess module."""
 
 import os
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)